src/app/ngrx/reducers/filter.reducer.ts

```diff
@@ -20,6 +20,13 @@ export function FilterReducer(state: FilterModel = initialState, action: ActionW
         // but we can and maybe should do it in action creators instead of components)
         newState.when = action.payload.filter.when;
       }
+      // the reason that there is a range separate from the standard range "when"
+      //  is because the when value becomes undefined when the begin and end dates do not
+      //  match one of the valid options; however, the range sticks around so that prev/next
+      //  buttons can still be used to page through based on the last selected range and the date pickers
```

Comment:
👍  v helpful comment

Reply:
(could be a reason to rename 'when' to something more specific to canned-option nature?)

Reply:
Heh, I feel like this is a bit of a surprise feature but still maybe better than disabling the prev and next when it's not a standard range.

Reply:
yeah, maybe "standardRange" instead of "when"? Let's see if this standard vs custom idea sticks.

Reply:
that sounds good!